pom.xml

```diff
@@ -241,6 +241,13 @@
       <artifactId>vertx-junit5</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jeasy</groupId>
+      <artifactId>easy-random-randomizers</artifactId>
+      <version>5.0.0</version>
```

Comment:
Like the other deps, the version number should be stored in a property that is then referenced here.

Reply:
Switched to version property def